.github/workflows/cd.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
     # Only run if Tupaia CI passed
     if: ${{ github.event.workflow_run.conclusion == 'success' }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v2
```

Comment:
Possibly out of scope, but this is up to v4 now too. ([Changelog](https://github.com/aws-actions/configure-aws-credentials/blob/main/CHANGELOG.md))

Reply:
Good thinking, might as well try this upgrade. I'll spin up a deployment first to see if CD passes on it